packages/react-native-sdk/docusaurus/docs/reactnative/06-advanced/06-deeplinking.mdx

```diff
@@ -1,194 +1,193 @@
+{
+ return [RCTLinkingManager application:application
+                  continueUserActivity:userActivity
+                    restorationHandler:restorationHandler];
+}
+```
 
-    ##### Adding Associated Domains Entitlement
+##### Adding Associated Domains Entitlement
 
-    Within your Signing & Capabilities section of Xcode, add the Associated Domains entitlement, thus giving your app the capability to support external domains for not only Universal Links, but web credentials too.
+Within your Signing & Capabilities section of Xcode, add the Associated Domains entitlement, thus giving your app the capability to support external domains for not only Universal Links, but web credentials too.
 
-    <img src={XCodeSigningCapabilities} />
+![XCode Signing Capabilities](../assets/06-advanced/06-deeplinking/xcode-signing-capabilitites.png)
 
-    To add Associated Domains support click on +(plus) button just below the `Signing & Capabilities` section near to `Capability`. Search for `Associated Domains` and drag and drop it to the background. This will add the Associated Domains tab.
+To add Associated Domains support click on +(plus) button just below the `Signing & Capabilities` section near to `Capability`. Search for `Associated Domains` and drag and drop it to the background. This will add the Associated Domains tab.
 
-    Within the top empty field that presents itself after adding this entitlement, click on +(plus) icon and insert `applinks:` followed by your domain name.
+Within the top empty field that presents itself after adding this entitlement, click on +(plus) icon and insert `applinks:` followed by your domain name.
 
-    _The `applinks`: prefix is a nod to Xcode that this domain is being used for Universal Linking, and is one of a few prefixes that Xcode will recognise depending on the technology you are implementing._
+_The `applinks:` prefix is a nod to Xcode that this domain is being used for Universal Linking, and is one of a few prefixes that Xcode will recognise depending on the technology you are implementing._
 
-    One you are done with this step, a `.entitlements` file will be created, inside `ios/YOUR_APP_NAME` folder.
+One you are done with this step, a `.entitlements` file will be created, inside `ios/YOUR_APP_NAME` folder.
 
-    Now we will add the AASA (Apple App Site Association) File to our web project. To do this, follow the steps below:
+Now we will add the AASA (Apple App Site Association) File to our web project. To do this, follow the steps below:
 
-    - Create a `apple-app-site-association` file inside `.well-known` directory on the top level of your domain, so that the file is hosted at `https://domain.name/.well-known/assetlinks.json`.
+- Create a `apple-app-site-association` file inside `.well-known` directory on the top level of your domain, so that the file is hosted at `https://domain.name/.well-known/assetlinks.json`.
```

Comment:
```suggestion
- Create a `apple-app-site-association` file inside `.well-known` directory on the top level of your domain, so that the file is hosted at `https://domain.name/.well-known/apple-app-site-association`.
```